### PR TITLE
Notification tweaks

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

**Comment by wski** [comment]
https://github.com/ozone-development/ozp-center/issues/535

**Comment by wski** [comment]
@clarkjefcoat @SOAP1914

File: app/js/components/NavBar/notificationsmodal.jsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                                  &nbsp;&nbsp;
+                                  <label for="exampleSelect1">From Sender: &nbsp;</label>
+                                  <select className="form-control" id="exampleSelect1">
+                                    <option>Apps Mall</option>
+                                  </select>
+                                </fieldset>
+                              </div>
+                            </div>
+                            */}
+
```
Comment (clarkjefcoat) [inline app/js/components/NavBar/notificationsmodal.jsx]: If we really don't need this then delete it.

**Comment by clarkjefcoat** [comment]
:+1:

File: app/js/components/NavBar/notificationsmodal.jsx (hunk 1 above)
Comment (wski) [inline app/js/components/NavBar/notificationsmodal.jsx]: I'm leaving this here for the future, once we have p2p notifications this will be needed.

**Comment by SOAP1914** [comment]
screen shot 2016-04-25 at 3 52 11 pm
screen shot 2016-04-25 at 3 52 27 pm

👍

**Pull request merged**